components/Navbar.vue

```diff
@@ -38,9 +38,9 @@
                 ref="mobilSearchRef"
                 hide-filter
                 class="is-flex-grow-1 mt-3" />
-              <b-button class="cancel-btn" @click="hideMobileSearchBar">
+              <a class="cancel-btn p-3" @click="hideMobileSearchBar">
```

Comment:
I think this will be better as a `button` rather than `a`

Other then that... lgtm